test/WebJobs.Script.Tests/Artifacts/ApiHub/ApiHubTestHelper.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.ApiHub.Sdk;
+using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.ApiHub.Sdk.Table.Internal;
+using Microsoft.Azure.WebJobs.Extensions.ApiHub.Common;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Artifacts.ApiHub
+{
+    public static class ApiHubTestHelper
+    {
+        public const string Key = "AzureWebJobsSql";
+        public const string DataSetName = "default";
+        public const string TableName = "SampleTable";
```

Comment:
Not an issue that will immediately impact us right now if we're just running with mocks, but if we intend to run against a real environment, this would end up having the same issues we're running into (and trying to fix) with other tests where concurrent test runs will step over each other. Ideally, this would be different for every test run and the test would take care of creating (and disposing of) the tables it needs.